### ISSUE TITLE
S3_TRUNCATE_KEYS is not working

### ISSUE THREAD

**Issue by drochgenius** [GREsau/localstack-persist#14]
# Context

* AWS S3 supports object key length up to 1024 characters.
* However, most OS like MacOS, have max file length of 255 chars.

Because of the way LocalStack persists S3 objects on disk, the key length is limited to 255 chars. When an S3 object key is longer than that is would error out:

```
exception while calling s3.PutObject: [Errno 36] File name too long: '/persisted-data/s3/assets/plankton-pipeline/your-very-long-s3-key'
```

# Problem

In LocalStack version 3.1.0, support for S3 keys longer than 255 chars was added via the **S3_TRUNCATE_KEYS** environment flag. 

https://github.com/localstack/localstack/issues/10047

However, even though `localstack-persist` is based on 3.6, this feature is not working.

**Comment by GREsau** [comment]
Thanks for the report - this should now be fixed in v3.6.1, which will be on docker hub in 5 mins or so 🙂

**Comment by drochgenius** [comment]
That's awesome! Thank you, I'll test it shortly.
